### ISSUE TITLE
How does this old version of scratchblocks compare to one up on the site?

### ISSUE THREAD

**Issue by ZenithRogue** [tjvr/wiki-scratchblocks#8]
I want to update the scratch wiki to have the new s3blocks format, but I am not quite clear on how this repo works. Do I just repoint the code to use the generated bundle.js that is used for s3block.github.io?

**Comment by Choco31415** [comment]
Hello Nitro! Sorry for not replying earlier; I don't check this repo often. :p 

To use this extension, all you need to do is put either or in LocalSettings.php:

```
wfLoadExtensions('ScratchBlocks');
require_once "$IP/extensions/ScratchBlocks/ScratchBlocks.php";
```

If you want to change the default languages supported, see `run_scratchblocks.js`.

Does this answer your question?

**Comment by ZenithRogue** [comment]
Hey, thanks for the reply! Unfortunately, I don't think this answers my question. The scratchblocks.php file has these lines of code:
```
$wgResourceModules['ext.scratchBlocks'] = array(
    'scripts' => array(
        'scratchblocks/src/scratchblocks.js',
        'scratchblocks/src/translations.js',
        'run_scratchblocks.js',
    ),
```
The main repo for scratchblocks builds a bundle.js file, and I am trying to figure out if i need to do any more than replace the aforementioned scratchblocks.js file with the new bundle.js.

**Comment by Choco31415** [comment]
I misread the original comment. Let's restart.

All that needs to happen is for 'scratchblocks/src/scratchblocks.js' and 'scratchblocks/src/translations.js' to be replaced with whatever files you need run before the page is loaded. Then, make sure either run_scratchblocks.js code (what replaces HTML with the pretty blocks) still works with the global JS functions available, or replace it with your own code. It's all around pretty flexible.

In other words, for the most part, it's just replacing the aforementioned file(s).
